code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -528,6 +542,7 @@
 	var/mob/living/carbon/human/owner
 	var/list/datum/spellbook_entry/entries = list()
 	var/list/categories = list()
+	var/owner_is_apprentice = 0 // to see if the book belongs to an apprentice
```

Comment:
The reason why I take this approach is so that the Non-Apprentice Wizard can't take the book for himself and profit off of 5 spell points for the cost of 2 and one saddened apprentice. This way they can't just take the book and hoard up infinite spell points.